tests/python_client/testcases/test_delete.py

```diff
@@ -1064,3 +1069,593 @@ def test_delete_sealed_only(self):
         collection_w.delete(expr)
 
         collection_w.query(expr, check_task=CheckTasks.check_query_empty)
+
+class TestDeleteString(TestcaseBase):
+    """
+    Test case of delete interface with string 
+    """
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_delete_entities_repeatedly_with_string(self):
+        """
+        target: test delete entities twice with string expr
+        method: delete with same expr twice
+        expected: No exception for second deletion
+        """
+        # init collection with nb default data
+        collection_w = self.init_collection_general(prefix, nb=tmp_nb, insert_data=True, primary_field=ct.default_string_field_name)[0]
+
+        # assert delete successfully and no exception
+        collection_w.delete(expr=default_string_expr)
+        collection_w.num_entities
```

Comment:
So no comparison is needed here after deletion? The function of calling "num_entities" is only flush?

Reply:
No call to num_entity should correspond to the growing segment, after the call to the sealed